microsoft/testsuites/mshv/mshv_root_tests.py

```diff
@@ -87,6 +88,13 @@ def verify_mshv_stress_vm_create(
             # fall back to defaults
             configs = [{}]
 
+        # This test can end up creating and a lot of ssh sessions and these kept active
+        # at the same time.
+        # In Ubuntu, the default limit is easily exceeded. So change the MaxSessions
+        # property in sshd_config to a high number that is unlikely to be exceeded.
+        node.tools[Ssh].set_max_session()
+        node.close()
```

Comment:
in set_max_session, it will restart sshd service, does it take effect? or we have to use node. Close()?

Reply:
Restarting sshd isn't sufficient. It doesn't take effect for existing connections. New connection needs to be established. Hence `node.close()`.  Similar usage is here: https://github.com/microsoft/lisa/blob/5f597d2b6c241278705f540b4a78fc2caa973e76/microsoft/testsuites/performance/common.py#L190-L195

Reply:
Can the set_max_session include this line directly?

Reply:
Done